src/BenchmarkDotNet/Toolchains/MonoWasm/WasmExecutor.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Helpers;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.Parameters;
+using BenchmarkDotNet.Toolchains.Results;
+using JetBrains.Annotations;
+
+namespace BenchmarkDotNet.Toolchains.MonoWasm
+
+{
+    [PublicAPI]
+    public class WasmExecutor : IExecutor
+    {
+        public WasmExecutor(string customDotNetCliPath) => CustomDotNetCliPath = customDotNetCliPath;
+
+        private string CustomDotNetCliPath { get; }
+
+        private string JavaScriptEngine = "v8";
```

Comment:
this field is not used anywhere. Should it be removed or should it be populated and used to allow for customization?